### DISCUSSION TITLE
how do i fix this? (i'm not on video rendering mode too btw)

### DISCUSSION THREAD

**Discussion by Lenancha** [JordanSantiagoYT/FNF-JS-Engine#373] [Q&A] [answered]
https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/83672015/2880c8d8-ea0c-4b65-b51e-dd7d313c5445

**Comment by JordanSantiagoYT** [comment]
when you tried loading a song the game did not find the song in the path it's supposed to be in, are you sure it's in mods/(your mod here)/songs/bite-fernan-mix?

the issue could also be that your song has spaces which you also added to the song file path

**Reply by Lenancha** [reply]
It is in the path it's supposed to be in idk why it always does that. (sorry for the late reply there's no way for me to get notifications on here since i'm not active here too much)

**Reply by JordanSantiagoYT** [reply]
let me see a ss of the path to the song folder, which file explorer shows you.

**Reply by Lenancha** [reply]
Alrighty (sorry for the late reply again) https://drive.google.com/file/d/116ej-q7UiC0D54A7JrYeje5tOYgmjFEG/view?usp=sharing

Edit: The zip file is trashed and everything is fixed.

**Reply by JordanSantiagoYT** [reply, marked as answer]
alright. it works for me.

the definitive issue ive found is you probably have THIS option turned off.
![image](https://github.com/user-attachments/assets/98867537-ccee-43d8-a9d3-39c2b340b047)

**Reply by Lenancha** [reply]
Ah I see now thank you so much!
